Model/Sync/Customers/Processor.php

```diff
@@ -249,6 +249,7 @@ public function processEntities($retryCustomerIds = [])
                 'response_code'
```

Comment:
### Issue

The customers sync LEFT JOINs _customers_ with _yotpo_customers_sync_.
For a large set of customers, this causes long execution times on top of the database, which leads the sync process to timeout.

###